ecosystem/typescript/sdk/src/providers/aptos_client.ts

```diff
@@ -233,6 +233,22 @@ export class AptosClient {
     return data;
   }
 
+  /**
+   * Queries a 0x1::object::Object resource associated with given address by resource type
```

Comment:
tbh, I am not completely sure what are the benefits with this function? it is exactly the same as `getAccountResource` but just that the first argument name is different (objectAddress vs accountAddress)

Reply:
@0xmaayan Yes, user can just use the `getAccountResource`  to get the object. Maybe no need to add this function. Just document somewhere to let user know how to get the object with AptosClient.

Reply:
agreed. maybe adding a test that "gets an object" so users would have an example on how they can get an object

Reply:
@0xmaayan changed. How could I request the CI?